docs/topics/clusterdefinitions.md

```diff
@@ -732,7 +732,7 @@ A cluster can have 0 to 12 agent pool profiles. Agent Pool Profiles are used for
 | extensions | no | Specifies list of extensions to enable for the agent profile. More details about [agentPoolProfiles extensions](extensions.md#agentpoolprofiles) |
 | preProvisionExtension | no | Specifies an extension to be run before the cluster is brought up. More details about [agentPoolProfiles extensions](extensions.md#agentpoolprofiles) |
 | sysctldConfig                    | no                        | Configure Linux kernel parameters via /etc/sysctl.d/. See `sysctldConfig` [below](#feat-sysctld-config)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
-
+| proximityPlacementGroup          | no                        | Specifies the resource id of the Proximity Placement Group (PPG) to use for this agentpool.  Please find more details about PPG in this [Azure blog](https://azure.microsoft.com/en-us/blog/introducing-proximity-placement-groups). Note that the PPG should be created in advance. The following [Azure CLI documentation](https://docs.microsoft.com/en-us/cli/azure/ppg?view=azure-cli-latest#az-ppg-create) explains how to create a PPG.                                                                                                                                                              |
```

Comment:
`proximityPlacementGroupID` is better IMO

Reply:
Based on the way the actual ARM spec is composed I agree with Andy:

E.g.,

```
    if masterProfile.ProximityPlacementGroup != "" {
 		ppg := masterProfile.ProximityPlacementGroup
 		vmProperties.ProximityPlacementGroup = &compute.SubResource{
 			ID: &ppg,
 		}
 	}
```

My only other thought: is there any chance that additional `ProximityPlacementGroup` properties will be exposed on the api model? Should we make the api model `ProximityPlacementGroup` config interface a struct? If not, then let's call it `ProximityPlacementGroupID`.

Reply:
@andyzhangx , @jackfrancis - proximityPlacementGroupID makes sense. Will update the PR.

@jackfrancis - as now don't anticipate any further properties being exposed.